actors/builtin/verifreg/verified_registry_test.go

```diff
@@ -325,7 +603,62 @@ func (h *verifRegActorTestHarness) removeVerifier(rt *mock.Runtime, verifier add
 	h.assertVerifierRemoved(rt, verifier)
 }
 
-func (h *verifRegActorTestHarness) getVerifier(rt *mock.Runtime, a address.Address) verifreg.DataCap {
+func (h *verifRegActorTestHarness) useBytes(rt *mock.Runtime, a address.Address, dealSize verifreg.DataCap) {
+	rt.ExpectValidateCallerAddr(builtin.StorageMarketActorAddr)
+	rt.SetCaller(builtin.StorageMarketActorAddr, builtin.StorageMinerActorCodeID)
+
+	// client will be removed if it's cap after the usebytes call is less than MinVerifiedDealSize
+	var removed bool
+	prev := h.getVerifiedClientCap(rt, a)
+	newCap := big.Sub(prev, dealSize)
+	if newCap.LessThan(verifreg.MinVerifiedDealSize) {
+		removed = true
+	}
```

Comment:
This is ok, but borderline too much logic/smarts in the harness. It's harder to tell what the test cases are actually covering, and if this changes we might accidentally lose coverage of some case that was implicit, rather than explicit.

Alternatively this could not make this check at all (but some call sites make it explicitly), or it could take a parameter being the expected cap after the bytes are used.